giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/config/SystemMessageHandlerConfig.java

```diff
@@ -0,0 +1,22 @@
+package edu.asu.diging.gilesecosystem.web.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import edu.asu.diging.gilesecosystem.septemberutil.service.impl.SystemMessageHandler;
+import edu.asu.diging.gilesecosystem.util.properties.IPropertiesManager;
+import edu.asu.diging.gilesecosystem.web.service.properties.Properties;
+
+@Configuration
+public class SystemMessageHandlerConfig {
```

Comment:
lets just have one configuration class for both message handler and rest template, no reason for so many